packages/mermaid/src/diagram-api/types.ts

```diff
@@ -16,11 +16,21 @@ export interface InjectUtils {
  * Generic Diagram DB that may apply to any diagram type.
  */
 export interface DiagramDB {
+  // config
+  resetConfig?: () => void;
+  setConfig?: (config: BaseDiagramConfig) => void;
+  getConfig?: () => BaseDiagramConfig | undefined;
```

Comment:
I don't think these functions should be added to the `DiagramDB` type (except maybe `getConfig()`.

```suggestion
```

If we do want to add them, we should add it in a separate PR.

Reply:
Removed for now, here is the changes so when we need to re-add it: https://github.com/mermaid-js/mermaid/compare/cb5f70c1396671d1570651b99b9ce79f27c60a2c...3f3a7340e3e282443290af61198f25fb3e6e1964